modules/nw-nodeport-service-range-edit.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-node-port-service-range.adoc
+
+[id="nw-nodeport-service-range-edit_{context}"]
+= Expanding the node port range
+
+You can expand the node port range for the cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To expand the node port range, enter the following command. Replace `<port>` with the largest port number in the new range.
++
+[source,terminal]
+----
+$ oc patch network.config.openshift.io cluster --type=merge -p \
+  '{
+    "spec":
+      { "serviceNodePortRange": "30000-<port>" }
+  }'
+----
++
+.Example output
+[source,terminal]
+----
+network.config.openshift.io/cluster patched
+----
+
+. To confirm that the configuration is active, enter the following command. It can take several minutes for the update to apply. The `service-node-port-range` entry in the ConfigMap appears only if the `serviceNodePortRange` field is set.
```

Comment:
Should this specify "... only if the `serviceNodePortRange` field is set in the network configuration." or something?

Reply:
@bergerhoffer, possibly. I thought it might be excessive given the context. Or it might be too much already.

The `service-node-port-range` will definitely always be set if someone completed the prior steps, the text is only meant to cover the case where someone runs it anyway out of order, but maybe that's irrelevant.

I think I'll just omit it.